packages/react-i18n/src/test/i18n.test.ts

```diff
@@ -1156,14 +1156,14 @@ describe('I18n', () => {
     });
 
     it('formats a date using DateStyle.Short', () => {
-      const date = new Date('2012-12-20T00:00:00-00:00');
+      const date = new Date('2012-12-09T00:00:00-00:00');
```

Comment:
nit: suffix ISO date string with `Z`

Reply:
There are like 20 instances of `-00:00` in this test file and I'd rather not switch the convention in this PR. Can be a separate quick PR to do that though.